lib/dynos.js

```diff
@@ -201,12 +205,41 @@ async function http_restart_dyno(pg_pool, req, res, regex) {
   return httph.ok_response(res, JSON.stringify({"status":"pending", type:dyno_info[0], dyno:dyno_info[1]}));
 }
 
+// public
+async function http_attach_dyno(pg_pool, req, res, regex) {
+  let app_key = httph.first_match(req.url, regex);
+  let dyno_key = httph.second_match(req.url, regex);
+  let app = await common.app_exists(pg_pool, app_key);
+  let payload = await httph.buffer_json(req);
+  let [dyno_type, dyno_id] = dyno_key.split('.');
+
+  // ensure the dyno exists before sending command
+  let release = null
+  try {
+    release = await common.latest_release(pg_pool, app.app_uuid)
+  } catch (e) {
+    throw new common.NotFoundError('The specified dyno was not found.');
+  }
+  let info = await dyno_info(pg_pool, app.app_uuid, app.app_name, app.space_name);
+  info = instances_to_payload(app.app_uuid, app.app_name, app.space_name, app.org_uuid, info, release);
+  if(info.filter((x) => x.type === dyno_type && x.name == dyno_id).length === 0) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/Akkeris/controller-api/pullRequest?prid=5050045)